Singer.API/Controllers/EventController.cs

```diff
@@ -259,7 +273,10 @@ public async Task<ActionResult<EventRegisterDetailsDTO>> GetEventRegisterDetails
             StartDateTime = x.StartDateTime,
             Registrations = registrations.Where(y => y.EventSlot.Id == x.Id).Select(z => new EventCareUserRegistrationDTO
             {
+               RegistrationId = z.Id,
```

Comment:
shouldn't we use a little bit more describing name then x, y, z? Most certainly when y and z are the same objects.

Reply:
agree

Reply:
Will fix.